lib/node_modules/@stdlib/stats/base/dists/wald/pdf/README.md

```diff
@@ -0,0 +1,293 @@
+# Probability Density Function
+
+> [Wald][wald-distribution] distribution probability density function (PDF).
+
+<section class="intro">
+
+The [probability density function][pdf] (PDF) for a [Wald][wald-distribution] random variable is
+
+<!-- <equation class="equation" label="eq:wald_pdf" align="center" raw="f(x;\mu,\lambda)=\sqrt{\frac{\lambda}{2\pi x^3}}\, e^{-\frac{\lambda(x-\mu)^2}{2\mu^2 x}}" alt="Probability density function (PDF) for a Wald distribution."> -->
+
+```math
+f(x;\mu,\lambda)=\sqrt{\frac{\lambda}{2\pi x^3}}\, e^{-\frac{\lambda(x-\mu)^2}{2\mu^2 x}}
+```
+
+<!-- <div class="equation" align="center" data-raw-text="f(x;\mu,\lambda)=\sqrt{\frac{\lambda}{2\pi x^3}}\, e^{-\frac{\lambda(x-\mu)^2}{2\mu^2 x}}" data-equation="eq:wald_pdf">
+    <img src="https://cdn.jsdelivr.net/gh/stdlib-js/stdlib@51534079fef45e990850102147e8945fb023d1d0/lib/node_modules/@stdlib/stats/base/dists/wald/pdf/docs/img/equation_wald_pdf.svg" alt="Probability density function (PDF) for a Wald distribution.">
+    <br>
+</div> -->
+
+<!-- </equation> -->
+
+where `µ > 0` is the mean and `λ > 0` is the shape parameter.
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var pdf = require( '@stdlib/stats/base/dists/wald/pdf' );
+```
+
+#### pdf( x, mu, lambda )
+
+Evaluates the [probability density function][pdf] (PDF) for a [Wald][wald-distribution] distribution with parameters `mu` (mean) and `lambda` (shape parameter).
+
+```javascript
+var y = pdf( 2.0, 1.0, 1.0 );
+// returns ~0.110
+
+y = pdf( 0.5, 2.0, 3.0 );
+// returns ~0.362
+```
+
+If provided `NaN` as any argument, the function returns `NaN`.
+
+```javascript
+var y = pdf( NaN, 1.0, 1.0 );
+// returns NaN
+
+y = pdf( 1.0, NaN, 1.0 );
+// returns NaN
+
+y = pdf( 1.0, 1.0, NaN );
+// returns NaN
+```
+
+If provided `mu <= 0`, the function returns `NaN`.
+
+```javascript
+var y = pdf( 2.0, 0.0, 1.0 );
+// returns NaN
+
+y = pdf( 2.0, -1.0, 1.0 );
+// returns NaN
+```
+
+If provided `lambda < 0`, the function returns `NaN`.
+
+```javascript
+var y = pdf( 2.0, 1.0, -1.0 );
+// returns NaN
+```
+
+If provided `lambda = 0`, the function evaluates the [PDF][pdf] of a [degenerate distribution][degenerate-distribution] centered at `mu`.
+
+```javascript
+var y = pdf( 2.0, 1.0, 0.0 );
+// returns 0.0
+
+y = pdf( 1.0, 1.0, 0.0 );
+// returns Infinity
+```
+
+If provided `x <= 0`, the function returns `0.0`.
+
+```javascript
+var y = pdf( 0.0, 1.0, 1.0 );
+// returns 0.0
+
+y = pdf( -1.0, 1.0, 1.0 );
+// returns 0.0
+```
+
+#### pdf.factory( mu, lambda )
+
+Partially applies `mu` and `lambda` to create a reusable `function` for evaluating the PDF.
+
+```javascript
+var mypdf = pdf.factory( 1.0, 1.0 );
+
+var y = mypdf( 2.0 );
+// returns ~0.110
+
+y = mypdf( 0.5 );
+// returns ~0.879
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var randu = require( '@stdlib/random/base/randu' );
```

Comment:
See https://github.com/stdlib-js/stdlib/pull/9439 for an example of how to replace manual `for` loops.